src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md

```diff
@@ -34,6 +34,23 @@
   `http.client.request.duration` metrics on .NET Framework for `HttpWebRequest`.
   ([#4870](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4870))
 
+* Following `HttpClient` metrics will now be enabled by default when targeting
+  `.NET8.0` framework.
+
+  * **Meter** : `System.Net.Http`
```

Comment:
as an end-user reading this, its not obvious what are these metrics. Its best to clarify that some of the metrics are part of the OTel Semantic Conventions (link to it). 

Also - does these metrics respect the OTEL_SEMCONV_STABILITY_OPT_IN flag? Please clarify that briefly in the changelog. (And of course Readme.md should have full details)

Reply:
Ideally, we would link all metrics to this https://github.com/open-telemetry/semantic-conventions/pull/283. We need to wait for that PR to merge first.

I will clarify the flag part.

Reply:
Clarified the flag part. PTAL. The PR is still not merged, will need to come back and update the links after the merge.

Reply:
One suggestion...

@noahfalk shared this doc with me: https://learn.microsoft.com/dotnet/core/diagnostics/built-in-metrics

We should probably link to it here and in aspnetcore instrumentation?

Reply:
This is great. I will fix the links. I was not aware of this one., was waiting on spec PR to merge. Thanks!